pkgs/development/compilers/aspectj/default.nix

```diff
@@ -1,12 +1,14 @@
 {stdenv, fetchurl, jre}:
 
 stdenv.mkDerivation rec {
-  name = "aspectj-1.5.2";
+  pname = "aspectj";
+  version = "1.9.6";
+  versionSnakeCase = builtins.replaceStrings ["."] ["_"] version;
```

Comment:
```suggestion
```

Use something like

```nix
  src = let
    versionSnakeCase = builtins.replaceStrings ["."] ["_"] version;
  in fetchurl {
    url = "http://archive.eclipse.org/tools/aspectj/${name}.jar";
    sha256 = "1b3mx248dc1xka1vgsl0jj4sm0nfjsqdcj9r9036mvixj1zj3nmh";
    url = "https://github.com/eclipse/org.aspectj/releases/download/V${versionSnakeCase}/aspectj-${version}.jar";
    sha256 = "02jh66l3vw57k9a4dxlga3qh3487r36gyi6k2z2mmqxbpqajslja";
  };
```

We should avoid adding unnecessary environment variables to the build environment.

Reply:
I have applied the suggestion.